### PR TITLE
Removing FAQ about MinGapLimit, it's changed in 1.1.10

### PR DESCRIPTION
**Pull request by PulpCattel**
Like [371](https://github.com/zkSNACKs/WasabiDoc/pull/371) this should wait until 1.1.10. 

It follows [2688](https://github.com/zkSNACKs/WalletWasabi/pull/2688), now that there's no more recycling, the corresponding FAQ could be removed. 
Instead, this adds a quick explanation about the new notification.

### CODE REVIEW

File: docs/FAQ/FAQ-UseWasabi.md [hunk 1]
```diff
@@ -257,28 +257,9 @@ Then you can click on `Generate` which will now show you the address, and immedi
...
-At `m/84'/0'/0'/0/20` (your 21st address), if you never used any of the previously generated addresses, Wasabi will start recycling the previous unused ones and will automatically overwrite the labels.
-E.g., Wasabi will not generate address `m/84'/0'/0'/0/21` with label: "address n. 22", instead it'll regenerate one between `m/84'/0'/0'/0/0` and `m/84'/0'/0'/0/20` and will overwrite previous label to "address n. 22".
-
-To increase the number of freshly new generated addresses, you have to increase the `MinGapLimit` json property of your `wallet.json` file.
 
-* Go to `File/Open/Wallets Folder` or navigate into the `Wallets` folder inside your [Wasabi data folder](/FAQ/FAQ-UseWasabi.md#where-can-i-find-the-wasabi-data-folder) and open your wallet file.
-* Close Wasabi Wallet.
-* Edit the `MinGapLimit` json property in the wallet file.
+If you generate more than 21 unused addresses, a warning will appear saying `MinGapLimit increased from 21 to 22.` and so on. 
+The MinGapLimit is the maximum number of consecutive unused addresses in your sequence of addresses, Wasabi automatically increases this value if needed when you generate new receiving addresses.
```
Comment (RiccardoMasutti) [inline docs/FAQ/FAQ-UseWasabi.md]:
```suggestion
The `MinGapLimit` is the maximum number of consecutive unused addresses in your sequence of addresses, Wasabi automatically increases this value if needed when you generate new receiving addresses. 
```

File: docs/FAQ/FAQ-UseWasabi.md [hunk 2]
```diff
@@ -257,28 +257,9 @@ Then you can click on `Generate` which will now show you the address, and immedi
...
-
-At `m/84'/0'/0'/0/20` (your 21st address), if you never used any of the previously generated addresses, Wasabi will start recycling the previous unused ones and will automatically overwrite the labels.
-E.g., Wasabi will not generate address `m/84'/0'/0'/0/21` with label: "address n. 22", instead it'll regenerate one between `m/84'/0'/0'/0/0` and `m/84'/0'/0'/0/20` and will overwrite previous label to "address n. 22".
-
-To increase the number of freshly new generated addresses, you have to increase the `MinGapLimit` json property of your `wallet.json` file.
 
-* Go to `File/Open/Wallets Folder` or navigate into the `Wallets` folder inside your [Wasabi data folder](/FAQ/FAQ-UseWasabi.md#where-can-i-find-the-wasabi-data-folder) and open your wallet file.
-* Close Wasabi Wallet.
-* Edit the `MinGapLimit` json property in the wallet file.
+If you generate more than 21 unused addresses, a warning will appear saying `MinGapLimit increased from 21 to 22.` and so on.
```
Comment (yahiheb) [inline docs/FAQ/FAQ-UseWasabi.md]:
```suggestion
If you generate more than 21 unused addresses, a notification will appear saying `MinGapLimit increased from 21 to 22.` and so on.
```
Or maybe a `warning notification`

File: docs/FAQ/FAQ-UseWasabi.md [hunk 3]
```diff
@@ -257,28 +257,9 @@ Then you can click on `Generate` which will now show you the address, and immedi
...
-At `m/84'/0'/0'/0/20` (your 21st address), if you never used any of the previously generated addresses, Wasabi will start recycling the previous unused ones and will automatically overwrite the labels.
-E.g., Wasabi will not generate address `m/84'/0'/0'/0/21` with label: "address n. 22", instead it'll regenerate one between `m/84'/0'/0'/0/0` and `m/84'/0'/0'/0/20` and will overwrite previous label to "address n. 22".
-
-To increase the number of freshly new generated addresses, you have to increase the `MinGapLimit` json property of your `wallet.json` file.
 
-* Go to `File/Open/Wallets Folder` or navigate into the `Wallets` folder inside your [Wasabi data folder](/FAQ/FAQ-UseWasabi.md#where-can-i-find-the-wasabi-data-folder) and open your wallet file.
-* Close Wasabi Wallet.
-* Edit the `MinGapLimit` json property in the wallet file.
+If you generate more than 21 unused addresses, a warning will appear saying `MinGapLimit increased from 21 to 22.` and so on. 
+The `MinGapLimit` is the maximum number of consecutive unused addresses in your sequence of addresses, Wasabi automatically increases this value if needed when you generate new receiving addresses.
```
Comment (yahiheb) [inline docs/FAQ/FAQ-UseWasabi.md]:
```suggestion
The `MinGapLimit` is the maximum number of consecutive unused addresses in your sequence of addresses, Wasabi automatically increases this value if needed when you generate new receiving addresses.
```

File: docs/FAQ/FAQ-UseWasabi.md (hunk 2 above)
Comment (PulpCattel) [reply docs/FAQ/FAQ-UseWasabi.md]: What about:
"a notification will appear saying `Warning! MinGapLimit increased from 21 to 22.`"

Comment (yahiheb) [reply docs/FAQ/FAQ-UseWasabi.md]: I like that yes.

**Pull request merged**